src/api/test/functional/interconnect_test.rb

```diff
@@ -203,8 +203,6 @@ def test_read_and_command_tests
     get '/source/RemoteInstance:BaseDistro/pack1?view=info&parse=1' # licensedigger needs it
     assert_response :success
     assert_xml_tag(tag: 'sourceinfo', attributes: { package: 'pack1' })
-    post '/source/RemoteInstance:BaseDistro/pack1', params: { cmd: 'showlinked' }
-    assert_response :success
```

Comment:
Before this just always answered with `<collection />` no matter if something linked to it or not. Now it will tell you that this won't work for remote projects...